client/my-sites/checkout/cart/secondary-cart.jsx

```diff
@@ -1,109 +0,0 @@
-/**
- * External dependencies
- */
-import PropTypes from 'prop-types';
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { localize } from 'i18n-calypso';
-import classNames from 'classnames';
-
-/**
- * Internal dependencies
- */
-import CartBody from './cart-body';
-import CartFreeUserPlanUpsell from 'calypso/my-sites/checkout/cart/cart-free-user-plan-upsell';
-import CartMessages from './cart-messages';
-import CartSummaryBar from './cart-summary-bar';
```

Comment:
import doesn't seem to be used elsewhere

Reply:
You're right. There's another one.